content/en/news/releases/1.13.x/announcing-1.13/change-notes/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Announcing Istio 1.13.0
+linktitle: 1.13.0
+subtitle: Minor Release
+description: Istio 1.13.0 minor release.
+publishdate: 2021-02-08
+release: 1.13.0
+aliases:
+    - /news/announcing-1.13.0
+---
+
+# Release Notes
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/about/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+## Traffic Management
+
+- **Added** an API (CRD) for configuring `ProxyConfig` values containing a stable subset of the configuration from `MeshConfig.DefaultConfig`.
+
+
+- **Promoted** WorkloadGroup to v1beta1.
+  ([Issue #25652](https://github.com/istio/istio/issues/25652))
+
+
+- **Improved** istio-agent health probe rewrite to not re-use connections, mirring Kubernetes' probing behavior.
+  ([Issue #36390](https://github.com/istio/istio/issues/36390))
+
+- **Improved** the default `PILOT_MAX_REQUESTS_PER_SECOND`, which limits the number of **new** XDS connections per second,
+to 25 (from 100). This has been shown to improve performance under high load.
+
+
+- **Updated** the control plane to read `EndpointSlice` insead of `Endpoints`
+for service discovery for Kubernetes 1.21 or later. To switch back to the old
+`Endpoints` based behavior set `PILOT_USE_ENDPOINT_SLICE=false` in istiod.
+
+
+- **Added** support for hostname-based multi-network gateways for east-west traffic. The hostname will be resolved in
+the control plane and each of the IPs will be used as an endpoint. This behaviour can be disabled by setting
+`RESOLVE_HOSTNAME_GATEWAYS=false` for istiod.  ([Issue #29359](https://github.com/istio/istio/issues/29359))
+
+- **Added** support rewriting gRPC probes
+
+- **Added** a feature flag `PILOT_LEGACY_INGRESS_BEHAVIOR`, default to false.
+If this is set to true, istio ingress will perform the legacy behavior,
+which does not meet https://kubernetes.io/docs/concepts/services-networking/ingress/#multiple-matches.
+  ([Issue #35033](https://github.com/istio/istio/issues/35033))
+
+- **Fixed** listeners to balance between Envoy worker threads. Fixes (#18152)[https://github.com/istio/istio/issues/18152).  ([Issue #18152](https://github.com/istio/istio/issues/18152))
```

Comment:
To have a consistent style, I suggest removing the part "Fixes (#18152)[https://github.com/istio/istio/issues/18152)." so that this item has the same style of other "Fixed" items (i.e., describe the fix and attach the issue number).

Reply:
Fixed.